seatunnel-connectors-v2/connector-influxdb/src/main/java/org/apache/seatunnel/connectors/seatunnel/influxdb/source/InfluxdbSourceReader.java

```diff
@@ -97,11 +102,26 @@ public void close() {
     }
 
     @Override
-    public void pollNext(Collector<SeaTunnelRow> output) {
-        while (!pendingSplits.isEmpty()) {
-            synchronized (output.getCheckpointLock()) {
-                InfluxDBSourceSplit split = pendingSplits.poll();
-                read(split, output);
+    public void pollNext(Collector<SeaTunnelRow> output) throws InterruptedException {
+        // reader influxDB By chunk
+        if (StringUtils.isEmpty(config.getSplitKey()) && config.getChunkSize() > 0) {
```

Comment:
I think this is not advisable, because I believe that chunked reading puts a lot of performance pressure on InfluxDB, and some users would still prefer to shift the pressure to SeaTunnel.

Reply:
> I think this is not advisable, because I believe that chunked reading puts a lot of performance pressure on InfluxDB, and some users would still prefer to shift the pressure to SeaTunnel.

Can the pressure on InfluxDB be resolved by reducing parallelism (e.g. setting parallelism to 1) or config speed limit config in env https://seatunnel.apache.org/docs/2.3.5/concept/speed-limit?

Reply:
The parallelism was originally 1; it has nothing to do with the speed of reading, but rather the consumption of InfluxDB's own chunk operation.

Reply:
> I think this is not advisable, because I believe that chunked reading puts a lot of performance pressure on InfluxDB, and some users would still prefer to shift the pressure to SeaTunnel.

+1. Make sense to me.